seed/challenges/03-back-end-development-certification/nodejs-and-expressjs.json

```diff
@@ -212,7 +212,7 @@
         "You can view this Node School module's source code on GitHub at <a href='https://github.com/azat-co/expressworks'>https://github.com/azat-co/expressworks</a>.",
         "Complete \"Hello World!\"",
         "Complete \"Static\"",
-        "Complete \"Jade\"",
+        "Complete \"Jade\" (Jade has been renamed to Pug after a copyright issue)",
```

Comment:
It should be enough just to mention the name change and leave out the bit about the copyright issue